lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -68,7 +68,8 @@ class ProgressTabView(RetrieveAPIView):
         enrollment_mode: (str) a str representing the enrollment the user has ('audit', 'verified', ...)
         grading_policy:
             assignment_policies: List of serialized assignment grading policy objects, each has the following fields:
-                num_droppable: (int) the number of assignments able to be dropped
+                num_droppable: (int) the number of lowest scored assignments that will not be counted towards the final grade
+                short_label: (str) the assignment
```

Comment:
"the assignment name" maybe? but fine regardless

Reply:
oops I fell asleep while typing!!!